server/src/client.ts

```diff
@@ -11,7 +11,7 @@ export interface Client {
 
 export const buildClient = (provider?): Client => {
   const etherProvider = provider ?? new ethers.providers.StaticJsonRpcProvider(config.provider)
-  const ews = new ethers.Contract(config.ewsContract, EWSAbi, etherProvider) as EWS
+  const ews = new ethers.Contract(config.ewsContract, EWSAbi, etherProvider) as unknown as EWS
```

Comment:
as BaseContract will do

Reply:
Instead of `unknown`, the type `ethers.BaseContract` should also work. Not very important here